File: ama/AstroWeather2/app/src/main/java/com/android/blawniczak/astroweather/domain/woeid_city/WoeidCity.java
package com.android.blawniczak.astroweather.domain.woeid_city;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class WoeidCity {

    @SerializedName("query")
    @Expose
    public Query query;

}

File: paiwjs/AstroWeather2/app/src/main/java/com/android/blawniczak/astroweather/domain/woeid_city/WoeidCity.kt
package com.android.blawniczak.astroweather.domain.woeid_city

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class WoeidCity {

    @SerializedName("query")
    @Expose
    var query: Query? = null

}
